common/djangoapps/third_party_auth/middleware.py

```diff
@@ -45,12 +47,11 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: d
         collection of user consent for sharing data with a linked third-party
         authentication provider.
         """
-        running_pipeline = request.session.get('partial_pipeline')
-
-        if not running_pipeline:
+        if not pipeline.running(request):
             return
 
         view_module = view_func.__module__
-        quarantined_modules = request.session.get('third_party_auth_quarantined_modules', None)
+        strategy = social_utils.load_strategy(request)
+        quarantined_modules = strategy.session_get('third_party_auth_quarantined_modules')
```

Comment:
@UmanShahzad I think that's a possibility, but unlikely. Either way is fine with me, I just didn't think it made sense to refactor this during this PR without any clear reason, which is why I was asking. The earlier approach was also fewer lines of code ;)